sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/CosmosQueryRequestOptions.java

```diff
@@ -40,6 +41,7 @@
     private Map<String, String> customOptions;
     private boolean indexMetricsEnabled;
     private boolean queryPlanRetrievalDisallowed;
+    private UUID correlationActivityId;
```

Comment:
We should use type `String` here, to be in consistent with ActivityId type, which is also String. 
`ActivityId` throughout the cosmos SDK is exposed as String.

Reply:
No - because the Rntbd protocol is inconsistent as well (Guid vs. String only for CorrelatedActivityId)